packages/lastrev-build-prefetch-content/src/buildTasks/writeRedirects.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import _ from 'lodash';
+import Contentful, { getGlobalSettings } from '@last-rev/integration-contentful';
+
+import writeFile from '../helpers/writeFile';
+import { CONTENT_DIR, REDIRECTS_FILE } from '../constants';
+import mkdirIfNotExists from '../helpers/mkDirIfNotExists';
+import { BuildTask } from '../types';
+
+type Redirect = {
+  from: string;
+  to: string;
+  status?: number;
+  force?: boolean;
+};
+
+const writeRedirectFile = async (redirects: string): Promise<void> => {
+  await writeFile(REDIRECTS_FILE, redirects);
+};
+
+const writeRedirects: BuildTask = async (buildConfig, { adapterConfig }): Promise<void> => {
+  const getSettings = buildConfig.useAdapter ? Contentful(adapterConfig).getGlobalSettings : getGlobalSettings;
```

Comment:
Since we are just parsing the response, I don't think we need to have both of these. If we don't need the adapter, we can just use the standard globalSettings request and parse it accordingly ({ fields : { redirects } }). Either way, whether you use the adapter or not, I would just stick to a single retrieval method. Because the line below (32) looks to me like it would fail if useAdapter != true...